src/libraries/Common/tests/System/IO/VirtualDriveHelper.Windows.cs

```diff
@@ -87,7 +87,7 @@ char GetNextAvailableDriveLetter()
                 List<char> existingDrives = DriveInfo.GetDrives().Select(x => x.Name[0]).ToList();
 
                 // A,B are reserved, C is usually reserved
-                IEnumerable<int> range = Enumerable.Range('D', 'Z' - 'D');
+                IEnumerable<int> range = Enumerable.Range((int)'D', 'Z' - 'D');
```

Comment:
```suggestion
                IEnumerable<int> range = Enumerable.Range<int>('D', 'Z' - 'D');
```

Reply:
Similar in other places. Explicit selecting the result type and leave it to the compiler to convert the arguments to it implicitly is less bug prone.